index.html

```diff
@@ -3074,97 +3078,30 @@ <h3>WoT Binding Templates</h3>
         over a suitable network protocol. WoT is tackling this
         variety through <a>Protocol Bindings</a>, which must meet a
         number of constraints (see <a href="#sec-protocol-bindings"></a>).
-      </p>
+      </p> -->
       <p> The non-normative <a>WoT Binding Templates</a> specification
         [[?WOT-BINDING-TEMPLATES]] provides a collection of
-        communication metadata blueprints that give guidance on
-        how to interact with different <a>IoT platforms</a>.
-        When describing a particular IoT device or service,
-        the <a>Binding Template</a> for the corresponding <a>IoT Platform</a>
-        can be used to look up the communication metadata that
-        must be provided in the <a>Thing Description</a> to
-        support that platform.
+        blueprints that give guidance on how to interact with different <a>Things</a> 
+        that use different <a>transfer protocols</a>, <a>content types</a> or that are 
+        part different <a>IoT platforms</a> (e.g., OCF, oneM2M, OMA LWM2M, OPC UA) 
```

Comment:
For me, these are just platforms disguised as standards. If Philips Hue was a standard, it would be still a platform for me. 
Leaving aside the controversial opinion, if I say IoT standards instead of platforms, then things like Zigbee also come into picture. What I want to say here are "things" that combine protocols with media types in a certain way. Another word can be thought of but I don't want to use the word standards so that we can have bindings for Philips Hue, Oracle, WebThings etc.

Reply:
I have changed to saying always platforms and standards to avoid any conflict